tokio/src/runtime/basic_scheduler.rs

```diff
@@ -518,12 +537,14 @@ where
             task.shutdown();
         }
 
-        // Release owned tasks
         unsafe {
+            // safety: the basic scheduler's local state will not be accessed
+            // from another thread. Since dropping the scheduler closes the
+            // remote queue, no tasks will be notified externally.
```

Comment:
I'm not sure I follow this comment?

Reply:
reworded in https://github.com/tokio-rs/tokio/pull/1892/commits/03357f1b78744fd4bf3f963859f8573c22b25d6e, hopefully it's clearer now?